code/lib/core-server/src/utils/StoryIndexGenerator.ts

```diff
@@ -18,7 +18,7 @@ import type {
   StoryId,
   StoryName,
 } from '@storybook/types';
-import { userOrAutoTitleFromSpecifier, sortStoriesV7 } from '@storybook/preview-api';
+import { userOrAutoTitleFromSpecifier, sortStoriesV7 } from '@storybook/preview-api'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
This explains core-server growing significantly, but it suggests that the code isn't tree-shaken properly still, because we're only importing 2 small utility functions here.

Reply:
I was hoping that by pre-bundling this it would make it so just these 2 tiny functions would be injected.
Granted, this is a node-only package right now..

Reply:
I wonder if i could move this code from preview-api to some other package

Reply:
I stopped tracking this, because cjs files don't get tree-shaken anyway.